packages/astro/src/default/pages/[...slug].astro

```diff
@@ -18,7 +18,7 @@ const { lesson, logoLink, navList, title } = Astro.props as Props;
 
 <Layout title={title}>
   <div id="previews-container"></div>
-  <main class="max-w-full flex flex-col h-screen overflow-hidden" data-swap-root>
+  <main class="max-w-full flex flex-col h-full overflow-hidden" data-swap-root>
```

Comment:
This fixed the issue of not seeing the buttons on the bottom of the page on mobile.